src/calculateRentalCost.js

```diff
@@ -5,6 +5,28 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const daylyRent = 40;
+  let amount = days * daylyRent;
+
+  if (days <= 3) {
+    amount = days * daylyRent;
+
+    // return amount;
+  }
+
+  if (days >= 3 && days < 7) {
+    amount = days * daylyRent - 20;
+
+    return amount;
```

Comment:
The condition 'days >= 3 && days < 7' can be simplified to 'days >= 3' because the return statement will exit the function before reaching the next condition for 7 or more days. Also, use constants for the number of days and the discount amount instead of hardcoding them.